pkg/testutils/testutils.go

```diff
@@ -208,3 +211,42 @@ func EnsureServiceHealthy(t testing.TB, grpcAddr, httpAddr string, transportCred
 		require.Equal(t, http.StatusOK, resp.StatusCode, "unexpected status code received from server")
 	}
 }
+
+// MustDefaultConfig returns default server config with the playground, tracing and metrics turned off.
+func MustDefaultConfig() *config.Config {
```

Comment:
FWIW I believe these actually belong in `internal/server/config`, not test utils. I think that TODO was outdated.

Anything that relates to config should be colocated in our config package.

Reply:
`MustDefaultConfig` and `MustDefaultConfigWithRandomPort` produce server config which is defined in `internal/server/config`, and this is irrespective of where they are used. We should keep code in related packages, because anyone wanting to deal with server config and config defaults presumably should be getting that from the `internal/server/config` package, not a test util package.

We don't put constructors in the packages that use them, we put them in the packages that define the symbol we're constructing, for example.

Reply:
OK, moved.